README.md

```diff
@@ -226,6 +330,46 @@ except for predicate1 and predicate2 that will be delegated to the object-based
 }
 ```
 
+#### Probability Generation Fragmentation Strategy
+
+A fragmentation strategy that ignore quads based on a probability.
```

Comment:
Does it really ignore quads? Or does it apply the given sub-strategy only with a given chance?